src/crypto/x509/pkix/pkix.go

```diff
@@ -177,6 +179,10 @@ func (n *Name) FillFromRDNSequence(rdns *RDNSequence) {
 					n.PostalCode = append(n.PostalCode, value)
 				}
 			}
+
+			if len(t) == 7 && t[0] == 1 && t[1] == 2 && t[2] == 840 && t[3] == 113549 && t[4] == 1 && t[5] == 9 && t[6] == 1 {
```

Comment:
Following suite is fine, but why not a compareOID function?

Reply:
Good call, done.